cl/users/email_handlers.py

```diff
@@ -215,3 +223,90 @@ def schedule_failed_email(recipient_email: str) -> None:
     :return: None
     """
     pass
+
+
+def convert_list_to_str(
+    email_list: Optional[Union[List[str], Tuple[str]]] = None
+) -> str:
+    """Function to convert a list or tuple of email addresses to a string,
+    we only support storing one email address in an Email object.
+    """
+    if email_list:
+        return email_list[0]
+    return ""
+
+
+def get_email_body(
+    message: Union[EmailMessage, EmailMultiAlternatives],
+    plain: str,
+    html: str,
+) -> List[str]:
+    """Function to retrieve html and plain body content of an email
+
+    :param message: the message to extract the body content
+    :param plain: the plain content type we are looking for
+    :param html: the html content type we are looking for
+    :return: plaintext_body and html_body list of strings
+    """
+    plaintext_body = ""
+    html_body = ""
+    for part in message.walk():
+        if part.get_content_type() == plain:
+            plaintext_body = part.get_payload()
+            if html_body:
+                break
+        if part.get_content_type() == html:
+            html_body = part.get_payload()
+            if plaintext_body:
+                break
+
+    return [plaintext_body, html_body]
+
+
+def store_message(
+    from_email: str,
+    to: Union[List[str], Tuple[str]],
+    cc: Optional[Union[List[str], Tuple[str]]] = None,
+    bcc: Optional[Union[List[str], Tuple[str]]] = None,
```

Comment:
if you don't care of it's a list or tuple, you can use [`collections.abc.Iterable`](https://docs.python.org/3/library/collections.abc.html#collections.abc.Iterable).

Reply:
Yes, you're right, changed for Iterable